runtime/oti/xhelpers.m4

```diff
@@ -85,16 +95,18 @@ define({FILE_START},{
 	.text
 })
 
-})	dnl OSX
-
-define({FILE_END})
+define({CALL_C_FUNCTION},$1)
```

Comment:
This is a truly terrible name for the macro, especially given the existing macro CALL_C_FUNC which actually does what its name suggests.  How about C_FUNCTION_SYMBOL?

Reply:
What we really need are two macros - one to name public symbols (so the xcinterp.m4 change works), and one to call public symbols (so you can add the _ or PLT).